connector/manifest.xml

```diff
@@ -10,24 +10,39 @@
     <vendor name="vendor"/>
     <driver name="driver"/>
     <customizations>
-      <customization name="CAP_SELECT_INTO" value="yes"/>
-      <customization name="CAP_SELECT_TOP_INTO" value="yes"/>
+      <!-- Metadata -->
+      <customization name="CAP_FAST_METADATA" value="yes"/>
+      <customization name="CAP_QUERY_WHERE_FALSE_METADATA" value="yes"/>
+      <!-- Temporary Tables -->
       <customization name="CAP_CREATE_TEMP_TABLES" value="no"/>
+      <customization name="CAP_SELECT_INTO" value="no"/>
+      <customization name="CAP_SELECT_TOP_INTO" value="no"/>
+      <!-- String Splits -->
+      <customization name="CAP_SUPPORTS_SPLIT_FROM_LEFT" value="no"/>
+      <customization name="CAP_SUPPORTS_SPLIT_FROM_RIGHT" value="no"/>
+      <!-- Initial SQL-->
+      <customization name="CAP_SUPPORTS_INITIAL_SQL" value="yes"/>
+      <!-- Query -->
+      <customization name="CAP_QUERY_ALLOW_JOIN_REORDER" value="yes"/>
+      <customization name="CAP_QUERY_BOOL_IDENTIFIER_TO_LOGICAL" value="yes"/>
```

Comment:
the documentation for this capability is phrased _very_ confusingly. The sentence as written in the docs suggests that "yes" means "don't wrap boolean predicates in an equality expression", but the naming of the capability seems to suggest the opposite

perhaps we could investigate to confirm one way or the other? I don't think it's a big enough deal to hold up this whole PR, but filing a ticket to investigate it might be warranted

Reply:
Created investigation ticket [SQL-751](https://jira.mongodb.org/browse/SQL-751)